Foundation/NSHost.swift

```diff
@@ -14,8 +14,19 @@ import Darwin
 import Glibc
 #endif
 
+import Foundation
 import CoreFoundation
 
+extension String {
+    public static func fromCString(_ cString: UnsafeMutablePointer<Int8>) -> String {
```

Comment:
This does not belong in NSHost and is tangential to the fixes listed for _resolveCurrent; additionally there does not seem to be any API for this publicly on Darwin; it was deprecated for good reason since it did not have a claimed failure potential.